metamorph/store/postgresql/postgres.go

```diff
@@ -360,14 +371,15 @@ func (p *PostgreSQL) GetUnmined(ctx context.Context, since time.Time, limit int6
 		,full_status_updates
 		,raw_tx
 		,locked_by
+		,retries
 		FROM metamorph.transactions
 		WHERE locked_by = 'NONE'
```

Comment:
I think we have to include here also those which are locked by this `metamorph`-pod, not?
```
WHERE (locked_by = 'NONE' OR locked_by = $6)
```
Where `$6` would be `p.hostname` on line 382 

That will ensure that only 1 metamorph pod is loading and retrying these unmined txs as they are set locked on line 485.

Reply:
Yes, that was my question. So that will guarantee to retry stalled transactions.

Reply:
In that case you can then on line 485 first check if the locked by field is equal to "NONE" in order to save unnecessary updates on the DB.

```
		if data.LockedBy == "NONE" {
			err = p.setLockedBy(ctx, data.Hash, p.hostname)
			if err != nil {
				return nil, err
			}
		}

```